src/k8s-engine/lima.ts

```diff
@@ -106,10 +108,14 @@ interface LimaListResult {
 
 const console = Logging.lima;
 const MACHINE_NAME = '0';
+const INTERFACE_NAME = 'rd0';
+const OLD_INTERFACE_NAME = 'lima0';
 const IMAGE_VERSION = '0.2.1';
 
 /** The root-owned directory the VDE tools are installed into. */
 const VDE_DIR = '/opt/rancher-desktop';
+const RUN_LIMA_LOCATION = '/private/var/run/rancher-desktop-lima';
+const LIMA_SUDOERS_LOCATION = '/private/etc/sudoers.d/rancher-desktop-lima';
```

Comment:
These should be documented? (At least, I'm pretty sure they're not "The root-owned directory the VDE tools are installed into"…)

Reply:
Not sure what you mean, but I added the requirements from the issue.